spec/defines/resource_stream_spec.rb

```diff
@@ -101,6 +101,12 @@
               value: 'spdy',
               match: %r{\s+listen\s+\[::\]:80 spdy;}
             },
+            {
+              title: 'should set resolver(s)',
+              attr: 'resolver',
+              value: ['8.8.8.8', '8.8.4.4'],
+              match: %r{\s+resolver\s+8.8.8.8 8.8.4.4;}
```

Comment:
I'm always a fan of using [RFC5735](https://tools.ietf.org/html/rfc5735) IPs. 
Name | Range
-- | --
TEST-NET-1 | 192.0.2.0/24
TEST-NET-2 | 198.51.100.0/24
TEST-NET-3 | 203.0.113.0/24

Reply:
i can't believe no one's tried to abolish and sell these yet